CHANGELOG.adoc

```diff
@@ -50,6 +50,7 @@ They will be only accessible through the model browser dialog.
 - https://github.com/eclipse-syson/syson/issues/112[#112] [services] Improves `getAllReachable()` util service, allowing to reduce the time being spent to retrieve the elements of a given type.
 - https://github.com/eclipse-syson/syson/issues/134[#134] [diagrams] Add FeatureTyping Edge tool, in the same way as other existing links such as Subclassification, Redefinition...
 - https://github.com/eclipse-syson/syson/issues/137[#137] [details] Add kind property from RequirementConstraintMembership on Constraint core properties.
+- https://github.com/eclipse-syson/syson/issues/147[#147] [diagrams] Refactor compartments of `RequirementDefinition` and `RequirementUsage` to better fit the specification and examples.
```

Comment:
Not sure if it should be `[diagrams]` or `[general-view, requirement-view]` (I don't know the formatting for entries related to multiple modules)

Reply:
I would say `diagrams` if it is related to several diagrams.